Eatery Blue/UI/Favoriting/FavoritesViewController.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  FavoritesViewController.swift
+//  Eatery Blue
+//
+//  Created by Peter Bidoshi on 10/11/24.
+//
+
+import EateryModel
+import UIKit
+
+class FavoritesViewController: UIViewController {
+
+    // MARK: - Properties (View)
+
+    private let favoriteEateriesView = EateryListView()
+    private let favoriteItemsView = FavoritesItemsView()
+    private let favoritesNavigationView = FavoritesNavigationView()
+    private let scrollView = UIScrollView()
+    private let stackView = UIStackView()
+
+    // MARK: - Properties (Data)
+
+    private var allEateries: [Eatery] = []
+    private var favoriteEateries: [Eatery] = []
```

Comment:
would it make sense to have favoriteEateries and favoriteItems as sets instead of arrays for efficiency? especially since i see .contains() is being called on it later on

Reply:
Yeah. It would be like marginally better because the number of favorite eateries is like limited to the total number of eateries on campus (like 30 or smth idk) so it would still just end up being o(1) for that lookup. I think favoriteItems might be best to have as a set tho